### ISSUE TITLE
Not able to run the web service

### ISSUE THREAD

**Issue by goodrahstar** [Conchylicultor/DeepQA#12]
Hi Conchylicultor,

I was trying to run the web interface but I am not able to run it successfully, any pointers for this ?

Thanks

**Comment by Conchylicultor** [comment]
Any error message ?

**Comment by goodrahstar** [comment]
After running **python3 manage.py runserver**

I am getting this message.

Django version 1.10.2, using settings 'chatbot_website.settings'
Starting Channels development server at http://127.0.0.1:8000/
Channel layer default (asgi_redis.core.RedisChannelLayer)
Quit the server with CONTROL-C.
2016-10-22 18:50:24,462 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
2016-10-22 18:50:24,463 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
2016-10-22 18:50:24,464 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
2016-10-22 18:50:24,464 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
2016-10-22 18:50:24,469 - INFO - server - Using busy-loop synchronous mode on channel layer

**Comment by Conchylicultor** [comment]
It seems fine for me. So what happens when you go to http://127.0.0.1:8000/ after launching `python3 manage.py runserver` ? What exactly does not work ?

Before the message you posted, you should get some message from TensorFlow, and should see this line.
```
WARNING: Restoring previous model from /home/.../DeepQA/save/model-server/model.ckpt
```

Otherwise, does the chatbot works with the command line interface ?

**Comment by goodrahstar** [comment]
Yes, chatbot is working fine in interactive mode. When i ping the url: http://127.0.0.1:8000/ nothing happens. No message is shown to me on browser or on terminal.

And i do get this restore message from tf 

WARNING: Restoring previous model from /home/ubuntu/workspace/save/model-server/model.ckpt
Model restored.

**Comment by Conchylicultor** [comment]
So when you go to http://127.0.0.1:8000/ with your browser, The page is infinitly loading, or does your browser show you an error message like `This site can’t be reached` ?

You can try changing the default port, like:
`python3 manage.py runserver 0.0.0.0:63333`

Otherwise, I think it's more a firewall issue, or something related.

**Comment by Conchylicultor** [comment]
Are you running it from docker or a virtual machine ? In that case, if you try to reach the webpage from your host machine, you have to open the ports of the virtual machine.

**Comment by goodrahstar** [comment]
I am using Cloud 9 workspace to deploy the code and it exposes https://deep-qa-goodrahstar.c9users.io:8080/ this url... 

And port 8080 is opened for this machine so I am not sure why I am not able to access it...

**Comment by goodrahstar** [comment]
It got resolved i used 0.0.0.0 instead of 127.0.0.1 and now its working.
Thanks for help @Conchylicultor :)
